openbrews/recipe/add/yeast.html

```diff
@@ -0,0 +1,40 @@
+<div class="list list-inset">
```

Comment:
For some reason the top border is not appearing in this inset list
![image](https://cloud.githubusercontent.com/assets/4491428/19151896/4a3f91a6-8b9d-11e6-901d-56edb7d49a9c.png)